src/ProjectNeon/Assets/Features/Combatants/MemberState.cs

```diff
@@ -46,6 +50,14 @@ public MemberState(IStats baseStats)
     public void GainPrimaryResource(int numToGive) => _counters[PrimaryResource.Name].ChangeBy(numToGive);
     private IResourceType PrimaryResource => ResourceTypes[0];
 
+    public void FeedOn(string feedType)
```

Comment:
The other three Phases aren't being cleared correctly when phase changes.

Reply:
@SilasReinagel Solved.